lib/rules/index.mjs

```diff
@@ -6,7 +6,7 @@ import importLazy from 'import-lazy';
 
 /** @type {import('stylelint')['rules']} */
 const rules = {
-	'alpha-value-notation': importLazy(() => require('./alpha-value-notation/index.js'))(),
+	'alpha-value-notation': importLazy(() => require('./alpha-value-notation/index.cjs'))(),
```

Comment:
question

Is this a typo? Don't you want `mjs`?

Reply:
Not a typo. We still need `.cjs` when using `require()`; if using requiring `.cjs`, then the following error occurs:

```
Error [ERR_REQUIRE_ESM]: require() of ES Module /Users/masafumi.koba/git/stylelint/stylelint/lib/rules/alpha-value-notation/index.mjs not supported.
```

Like #7184, if we could replace `import-lazy` with a Promise-based code, we could rewrite `.cjs` to `.mjs` in `rules/index.mjs`.